app/src/main/res/values/strings.xml

```diff
@@ -156,4 +155,11 @@
     <string name="copy_message_fetch">Copying message to clipboard</string>
     <string name="copy_failed">Copying to clipboard failed</string>
     <string name="try_again_error_msg">Something went wrong, try again</string>
+    <string name="cancel">Cancel</string>
+    <string name="message_here">Message Here</string>
+    <string name="sender_name">Sender name</string>
+    <string name="social">Social</string>
+    <string name="my_stream">MyStream</string>
```

Comment:
There are some placeholders as well, categorize them by moving them at the bottom + adding comments and add a prefix or suffix _placeholder